### PR TITLE
feat: Add event bus

### PR DESCRIPTION
**Pull request by gfyrag**
Add event bus to trigger events inside or outside the service.

This PR add an interface ledger.Monitor which can be implemented to watch ledger events (writes).
Following this addition, it adds a ledger.Monitor implementation able to send event to an event bus.
The event bus can be internal (use plain old go channel) or external (the library used is able to connect to several brokers).
Actually, the only plugged broker is kafka.

### CODE REVIEW

**Review by flemzord — APPROVED** [review]

File: _health [hunk 1]
```diff
@@ -0,0 +1 @@
+{"postgres":"OK"}
```
Comment (altitude) [inline _health]: Not sure this is intentional

File: examples/publisher-http/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+volumes:
+  postgres:
+services:
+  postgres:
+    extends:
+      file: ../../docker-compose.yml
+      service: postgres
+  listener:
```
Comment (altitude) [inline examples/publisher-http/docker-compose.yml:9]: echoer?

**Review by altitude — APPROVED** [review]

File: _health (hunk 1 above)
Comment (gfyrag) [reply _health]: Oups...

File: examples/publisher-http/docker-compose.yml (hunk 2 above)
Comment (gfyrag) [reply examples/publisher-http/docker-compose.yml:9]: Sorry, renaming error. I fix.

**Pull request merged**